common/men_at_arms_types/00_maa_types.txt

```diff
@@ -423,6 +423,35 @@ massive_aerial_unit = {
 	
 }
 
+riflemen = {
+	type = archers
+	
+	damage = 65
+	toughness = 20
+	pursuit = 20
```

Comment:
They have the same pursuit as heavy cavalry. I would either make them 0 pursuit like other ranged units or give some really small one.